packages/rs-dapi-client/src/request_settings.rs

```diff
@@ -21,7 +21,8 @@ pub struct RequestSettings {
     pub connect_timeout: Option<Duration>,
     /// Timeout for a request.
     pub timeout: Option<Duration>,
-    /// Number of retries until returning the last error.
+    /// Number of retries until returning the last error. `0` means no retries, that is only one request will be sent.
```

Comment:
Number of retries in case of failed requests. If max retries reached, the last error is returned.